client/src/annoMatrix/annoMatrix.ts

```diff
@@ -567,6 +568,20 @@ Return cache keys for columns associated with this query.  May return
     query: Query
   ): Promise<[WhereCache | null, Dataframe]>;
 
+  _getViewOf(): AnnoMatrix {
+    /**
```

Comment:
super tiny nit:  tsdocs expects the method comments (ie, @params, @returns) to be placed before (outside) of the method.  See example at top of this page: https://tsdoc.org/

Reply:
Update [annoMatrix.ts::_getViewOf() tsdocs](https://github.com/chanzuckerberg/cellxgene/blob/mim/annoMatrix/client/src/annoMatrix/annoMatrix.ts#L571).